### ISSUE TITLE
Links green color contrast ratio is very low

### ISSUE THREAD

**Issue by mshaaban0** [fastify/website-metalsmith#136]
## 🐛 Bug Report
Links green colour contrast ratio doesn't meet WCAG guidelines, and it's really burning my eyes. [Color Contrast Checker](https://webaim.org/resources/contrastchecker/?fcolor=00D1B2&bcolor=FFFFFF)

<img width="743" alt="Screenshot 2019-10-24 at 10 47 44" src="https://user-images.githubusercontent.com/6163988/67469427-7b69f600-f64c-11e9-9763-1f65ba5c138d.png">

## To Reproduce

<img width="481" alt="Screenshot 2019-10-24 at 10 52 22" src="https://user-images.githubusercontent.com/6163988/67469365-5ecdbe00-f64c-11e9-82c6-4cbfcc856f13.png">

## Expected behavior

Colour contrast ratio must be at least 4.5:1 or more. Darkening the green color to `#008571` pass the guidelines and looks much better for me.

<img width="477" alt="Screenshot 2019-10-24 at 10 56 12" src="https://user-images.githubusercontent.com/6163988/67469731-e6b3c800-f64c-11e9-97ee-7cd71962a5c2.png">

<img width="728" alt="Screenshot 2019-10-24 at 10 54 39" src="https://user-images.githubusercontent.com/6163988/67469573-b0764880-f64c-11e9-935e-052d67bd05a8.png">

Happy to help with the issue if you guys don't mind the new color.

**Comment by mcollina** [comment]
Thanks for reporting!

@lmammino @delvedor wdyt?

**Comment by delvedor** [comment]
I agree that this can (and should) be improved!

**Comment by lmammino** [comment]
Hello @mshaaban0, thanks a lot for this detailed issue. This is an extremely important point where I should have done a better job. Thanks for highlighting this and, please, feel free to bring more of these accessibility issues to the table. I'll follow on with a PR to address this shortly!

**Comment by mshaaban0** [comment]
Thanks for taking care of this @lmammino. Sure, I'll keep my eyes opened for other issues. FYI [Tota11y](https://khan.github.io/tota11y/) (when it works) is a cool tool to identify high-level accessibility problems.

**Comment by lmammino** [comment]
Awesome, thanks a lot! I'll try to give it a go, to see if there are other major issues!
